wolfssl/wolfcrypt/cmac.h

```diff
@@ -24,9 +24,10 @@
 #define WOLF_CRYPT_CMAC_H
 
 #include <wolfssl/wolfcrypt/types.h>
-#include <wolfssl/wolfcrypt/aes.h>
 
-#if !defined(NO_AES) && defined(WOLFSSL_CMAC)
+#ifdef WOLFSSL_CMAC
+
+#include <wolfssl/wolfcrypt/aes.h>
```

Comment:
it is a header include, so most likely innocuous, but do we want to be guarding the inclusion of `aes.h` with 
`!NO_AES`?

Reply:
Yeah, I had that in there and removed looking at a few other spots.  The first thing it does is check, but I think you are right.  I'll update.